types/global.d.ts

```diff
@@ -55,6 +55,13 @@ declare module 'hardhat/config' {
 
   function extendEnvironment(extender: EnvironmentExtender): void;
 
+  interface HardhatConfig {
+    fireblocks?: FireblocksProviderConfig;
```

Comment:
I think we need to import `FireblocksProviderConfig` otherwise I believe this gets typed as any